### ISSUE TITLE
Fix Modal Focus

### ISSUE THREAD

**Issue by amalyam** [seas-computing/mark-one#229]
### Description

When a modal is open, it currently allows the user to tab into the browser's URL bar. For accessibility purposes, when a modal is open, keyboard tabbing should be restricted to only the content within the modal. The user should NOT be able to tab into the URL bar if a modal is active.

### Acceptance Criteria

- [ ] When the modal component is active, the user should not be able to move focus outside of the modal

### Additional Information

[W3C Modal Dialog Example](https://www.w3.org/WAI/ARIA/apg/patterns/dialog-modal/examples/dialog/)
[W3C Dialog (Modal) Pattern](https://www.w3.org/WAI/ARIA/apg/patterns/dialog-modal/), with more details about the focus when the dialog is closed.
